convert2rhel/unit_tests/main_test.py

```diff
@@ -137,6 +141,8 @@ def test_show_eula_nonexisting_file(self):
         "restore",
         unit_tests.CountableMockObject(),
     )
+    @unit_tests.mock(cert.SystemCert, "_get_cert", GetCertMocked())
```

Comment:
```suggestion
    @unit_tests.mock(cert.SystemCert, "_get_cert", lambda: ("anything", "anything"))
```

And you can remove `class GetCertMocked` ;)